framework/src/fvkernels/FVDiffusion.C

```diff
@@ -49,12 +49,16 @@ FVDiffusion::computeQpResidual()
 
   // If we are on internal faces, we interpolate the diffusivity as usual
   if (_var.isInternalFace(*_face_info))
-    interpolate(_coeff_interp_method,
-                coeff,
-                _coeff(elemArg(), state),
-                _coeff(neighborArg(), state),
-                *_face_info,
-                true);
+  {
+    const ADReal coeff_elem = _coeff(elemArg(), state);
+    const ADReal coeff_neighbor = _coeff(neighborArg(), state);
+    // If the diffusion coefficients are zero, then we can early return 0 (and avoid warnings if we
+    // have a harmonic interpolation)
+    if (!coeff_elem.value() && !coeff_neighbor.value())
```

Comment:
I think 1/(1/a+1/b) will have issues if any of them is 0. I had error messages for these scenarios plus cases when you have different signs. Those seemed to be too restrictive when it came to convergence (early in the nonlinear iteration we may hit states which don't satisfy these) so they got removed. I think we have the solution invalidate capabilities for this now.

Reply:
I know this PR is not ready yet, I just saw that this mentioned an issue I was active on. So this is just a comment.

Reply:
Yes there is an issue with that particular interpolation method when one of them is 0. But with respect to the physics we can only safely early return 0 if the diffusion coefficient is 0 on both sides. Additionally, uniform 0 is an actual user case where they've just specified a constant 0. I would bet that 0 on just one side is going to be pretty uncommon and if it happens then the warning out of `MathFVUtils` is great